demo/vl-tabs.html

```diff
@@ -27,6 +26,10 @@
       <vl-tabs-pane data-vl-id="fiets" data-vl-title="Fiets">
         Duis mollis, est non commodo luctus, nisi erat porttitor ligula, eget lacinia odio sem nec elit. Aenean eu leo quam. Pellentesque ornare sem lacinia quam venenatis vestibulum. Cras justo odio, dapibus ac facilisis in, egestas eget quam. Praesent commodo cursus magna, vel scelerisque nisl consectetur et.
       </vl-tabs-pane>
+      <vl-tabs-pane data-vl-id="auto">
+        <span slot="auto-title-slot"><b>Auto</b></span>
+        Duis vitae magna vitae eros pretium porttitor id quis mauris. Sed imperdiet a diam in suscipit. Nunc consectetur urna nunc, eu tempor odio rutrum non. Sed in sem convallis, placerat nisi nec, placerat velit. In fringilla ex sed malesuada dictum. Sed congue neque orci, quis porta mi tempus ultrices. Vivamus gravida magna eu enim aliquet, sit amet tempus ante pellentesque. Donec eget hendrerit odio, eget aliquam felis. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras vel maximus orci.
+      </vl-tabs-pane>
```

Comment:
Idem opmerking vl-tabs-base

Reply:
heb de tab met title slot toch behouden in deze vl-tabs.html demo omdat deze ook gebruikt wordt op de demo pagina van het departement omgeving (de andere demo pagina's zie ik daar niet). Heb wel een extra test case geschreven die specifiek de vl-tabs-title-slot.html pagina test

Reply:
als je daar deze mee bedoelt 
https://webcomponenten.omgeving.vlaanderen.be/demo/vl-tabs.html

die wordt gepubliceerd van zodra er een nieuwe release komt. dus die zal bevatten wat jij er hier nu van maakt ...